### PR TITLE
Feature: New version docs for BNA

### PR DESCRIPTION
**Pull request by nnkogift**
- Added v1.4.0 docs config to mkdocs.yml
 - Changed single page doc config for BNA to point to latest version
 - Fixed a typo for scorecard app single page doc config

### CODE REVIEW

**Comment by nnkogift** [comment]
@Philip-Larsen-Donnelly  we have a new version of the BNA app and we've added the configuration through this pull request. Kindly review it. 

Thanks!

File: mkdocs.yml [hunk 1]
```diff
@@ -409,6 +409,22 @@ nav:
...
+                  Configuration:
+                  - Interventions and indicators configuration: '@github(hisptz/unicef-apps-docs, src/commonmark/en/content/bna_v1.4.0/interventions-and-indicators-configuration.md, master)'
+                  - Map configuration: '@github(hisptz/unicef-apps-docs, src/commonmark/en/content/bna_v1.4.0/maps-configuration.md, master)'
+                  - Access: '@github(hisptz/unicef-apps-docs, src/commonmark/en/content/bna_v1.4.0/grant-access-to-user.md, master)'
+                  Usage:
+                  - Data analysis: '@github(hisptz/unicef-apps-docs, src/commonmark/en/content/bna_v1.4.0/data-analystics.md, master)'
+                  - BNA layout: '@github(hisptz/unicef-apps-docs, src/commonmark/en/content/bna_v1.4.0/bna-layout.md, master)'
+                  - BNA analysis and interpretation: '@github(hisptz/unicef-apps-docs, src/commonmark/en/content/bna_v1.4.0/bna-analysis-and-interpretation.md, master)'
+                  - Map view: '@github(hisptz/unicef-apps-docs, src/commonmark/en/content/bna_v1.4.0/map-view.md, master)'
+
```
Comment (Philip-Larsen-Donnelly) [inline mkdocs.yml:427]: The paths here are slightly wrong and that causes the build to fail
The folder seems to be bna_v.1.4.0 in your repository, not bna_v1.4.0 (note the period after the `v`)

File: mkdocs_single_page.yml [hunk 2]
```diff
@@ -368,8 +368,8 @@ nav:
         DHIS2 Android App: '@github(dhis2/dhis2-android-docs, DHIS2_Android_App_INDEX_singlepage.yml, main)' #Android uses external INDEX files
         Optional Apps:
           Action Tracker App: '@github(hisptz/unicef-apps-docs, src/commonmark/en/dhis2_action_tracker_manual_INDEX.md, master)'
-          Bottleneck Analysis App: '@github(hisptz/unicef-apps-docs, src/commonmark/en/dhis2_bottleneck_analysis_manual_INDEX.md, master)'
-          Interactive Scorecard Ap Manual: '@github(hisptz/unicef-apps-docs, src/commonmark/en/dhis2_scorecard_manual_INDEX.md, master)'
+          Bottleneck Analysis App: '@github(hisptz/unicef-apps-docs, src/commonmark/en/dhis2_bottleneck_analysis_manual_v.1.4.0_INDEX.md master)'
```
Comment (Philip-Larsen-Donnelly) [inline mkdocs_single_page.yml:371]: You are missing a comma here
```suggestion
          Bottleneck Analysis App: '@github(hisptz/unicef-apps-docs, src/commonmark/en/dhis2_bottleneck_analysis_manual_v.1.4.0_INDEX.md, master)'
```

**Comment by Philip-Larsen-Donnelly** [comment]
The PR check/preview doesn't work from a fork, so I will just merge and we can deal with any issues afterwards.

**Pull request merged**